range.c

```diff
@@ -982,11 +982,15 @@ range_each(VALUE range)
 
 /*
  *  call-seq:
- *     rng.begin    -> obj
+ *    begin -> object
  *
- *  Returns the object that defines the beginning of the range.
+ *  Returns the object that defined the beginning of +self+,
+ *  if one was given; otherwise returns +nil+:
```

Comment:
I would drop this line.  `nil` is the implicit beginning of unbounded ranges: `(..1) == (nil..1) # => true`.

Reply:
Fixed.